app/assets/javascripts/application_legacy.js

```diff
@@ -99,7 +99,9 @@ $(function() {
     // Submission will be done after 500ms of not typed, unless data-submit-onchange=changed,
     // in which case it happens when the input box loses its focus ('changed' event).
     // (changeDate is for bootstrap-datepicker)
-    $(document).on('changed keyup focusin changeDate', 'form[data-submit-onchange] input[type=number]:not([data-ignore-onchange])', function(e) {
+    $(document).on('changed keyup focusin changeDate',
+    'form[data-submit-onchange] input[type=text]:not([data-ignore-onchange]), form[data-submit-onchange] input[type=number]:not([data-ignore-onchange])', function(e) {
+        console.log("foo");
```

Comment:
With less code & removing the `console.log`:

```suggestion
        $(document).on('changed keyup focusin changeDate', 'form[data-submit-onchange] input[type=number][type=text]:not([data-ignore-onchange])', function(e) {
```